proposed 2.16.3 changelog contents

mikehardy opened the pull request:
If approved, this will be the changelog baked into the 2.16.3 release

BrayanDSO commented on changelog.asc:
```diff
@@ -2,6 +2,21 @@
+* Fix crash displaying whiteboard menu
+* Allow long preference titles to wrap
+* Allow 3rd party cookies in webview to match Anki behavior
+* Improve "add shortcut" icon
```
```suggestion
* Improve "add" shortcut icon
```

mikehardy replied:
I removed the quotes entirely based on this suggestion

BrayanDSO commented on changelog.asc:
```diff
@@ -2,6 +2,21 @@
+* Allow long preference titles to wrap
+* Allow 3rd party cookies in webview to match Anki behavior
+* Improve "add shortcut" icon
+* Sync community-provided translations from https://crowdin.com/project/ankidroid
```
nit: don't follow if you don't want, but some people may not understand what `Sync` would mean here
```suggestion
* New community-provided translations from https://crowdin.com/project/ankidroid
```

mikehardy replied:
I like that better, thank you

snowtimeglass commented:
Won't the merge of [#14234](https://github.com/ankidroid/Anki-Android/pull/14234) be ready in time for release of 2.16.3?

I have the same impression as dae does. 

>There have been quite a few reports of this, so it might be worth addressing in a 2.16.3 before switching over to a 2.17 branch.

[#14225](https://github.com/ankidroid/Anki-Android/issues/14225)

lukstbit commented:
The changelog should mention the removal of the Beolingus feature in  [#14181](https://github.com/ankidroid/Anki-Android/pull/14181)

mikehardy commented:
- added content based on my sweep through the PR queue just now
- and I fixed the date in the header
- organized into general structure of "announce+thanks" / "crash fixes" / "general fixes" / "other changes" as I think crashes are usually the highlight
 
re-pushed
